examples/full-jaffle-shop-demo/dbt/models/events.sql

```diff
@@ -1 +1 @@
-SELECT * FROM {{ ref('raw_events') }}
+SELECT *, 'https://placehold.co/300x100?text=id%20' || event as image_url  FROM {{ ref('raw_events') }}
```

Comment:
adds full image URL as a column in sql. So we can test it directly with dimensions, not just with additional dimensions